index.html

```diff
@@ -90,9 +90,24 @@
           date: "..",
           publisher: "..."
         },
-        "NMEA": {
-          title: "National Marine Electronics Association",
-          href: "https://www.nmea.org"
+        "HIPAA": {
```

Comment:
Why did you remove NMEA?

Reply:
It actually was replaced by an updated reference in my other PR.  However, the name was changed to refer to the actual specification rather than just the org (the only NMEA spec in wide use in IoT is that used to get data from GPS devices), so I'll search for other usages and repair them.